packages/bee-q/src/components/breadcrumb/_storybook/bq-breadcrumb.stories.tsx

```diff
@@ -0,0 +1,81 @@
+import type { Args, Meta, StoryObj } from '@storybook/web-components';
+
+import { html } from 'lit-html';
+
+import mdx from './bq-breadcrumb.mdx';
+
+const meta: Meta = {
+  title: 'Components/Breadcrumb',
+  component: 'bq-breadcrumb',
+  parameters: {
+    docs: {
+      page: mdx,
+    },
+  },
+  argTypes: {
+    text: { control: 'text', table: { disable: true } },
+    ['separator-icon']: { control: 'text' },
+  },
```

Comment:
Please check other components storybook examples and see how events are also added as ArgTypes 😉